pipeline_dp/budget_accounting.py

```diff
@@ -300,15 +300,15 @@ def __init__(self,
             total_delta: delta for the entire pipeline.
             num_aggregations: number of DP aggregations in the pipeline for
              which  'self' manages the budget. All aggregations should have
-             'budget_weight' = 1. When specified all aggregations will have
+             'budget_weight' = 1. When specified, all aggregations will have
              equal budget. It is useful to ensure that the pipeline has fixed
              number of DP aggregations.
             aggregation_weights: 'budget_weight' of aggregations for which
              'self' manages the budget. It is useful to ensure that the pipeline
               has a fixed number of DP aggregations with fixed weights.
 
         if num_aggregations and aggregation_weights are not set, there are no
```

Comment:
We need to make the same change for PLD Accountant below.

Also, I see many references to `budget_weight` in this file (for example a couple lines above). Maybe they could all be `budget weights` or `aggregation weights`?

Reply:
Thanks, PLDAccountant is udpated

`budget_weight` is the name for the [field](https://github.com/OpenMined/PipelineDP/blob/784961601d26afa0a08beedc7fe15587c3da9987/pipeline_dp/aggregate_params.py#L79) in `AggregateParams`